examples/mail/mail.php

```diff
@@ -57,7 +57,7 @@
   "content": [
     {
       "type": "text/html", 
-      "value": "<html><p>Hello, world!</p><img src=[CID GOES HERE]></img></html>"
+      "value": "<html><p>Hello, world!</p><img src=[CID GOES HERE]></html>"
```

Comment:
Should it have an internal forward slash, like so:
```
      "value": "<html><p>Hello, world!</p><img src=[CID GOES HERE] /></html>"
```

Reply:
xhtmlish yes, html5 not necessary.

Reply:
/ are actually invalid, not facultative, in HTML 5. It's an XML only feature, and so only used on the old XHTML standard.